tests/state/ViewerState/threshold_slice.test.ts

```diff
@@ -99,4 +113,29 @@ describe("ThresholdSlice", () => {
       expect(result.current.inRangeLUT).to.deep.equal(EXPECTED_IN_RANGE_LUT);
     });
   });
+
+  describe("serializeThresholdSlice", () => {
+    it("serializes thresholds", () => {
+      const { result } = renderHook(() => useViewerStateStore());
+      act(() => {
+        result.current.setThresholds(DEFAULT_THRESHOLDS);
+      });
+      expect(result.current.thresholds).to.deep.equal(DEFAULT_THRESHOLDS);
+      const serialized = serializeThresholdSlice(result.current);
+      expect(serialized).to.deep.equal({ [UrlParam.THRESHOLDS]: DEFAULT_THRESHOLDS_SERIALIZED });
+    });
+  });
+
+  describe("loadThresholdSliceFromParams", () => {
+    it("deserializes thresholds", () => {
+      const { result } = renderHook(() => useViewerStateStore());
+      const params = new URLSearchParams();
+      params.set(UrlParam.THRESHOLDS, DEFAULT_THRESHOLDS_SERIALIZED);
+      act(() => {
+        result.current.setThresholds([]);
+        loadThresholdSliceFromParams(result.current, params);
+      });
+      expect(result.current.thresholds).to.deep.equal(DEFAULT_THRESHOLDS);
+    });
+  });
```

Comment:
Unit tests here are relatively minimal since I test most of the edge cases in `url_utils.test.ts`